components/table/table.jsx

```diff
@@ -287,15 +273,30 @@ class InfiniteTable extends React.PureComponent {
 
                     return <TableRow key={row.id} {...props} />
                   })}
-
-                {isFirstPageLoaded && !isLastPageLoaded && (
-                  <LoadMoreRow
-                    observe={showNextLoad}
-                    onVisible={this.loadNextPage}
-                  />
-                )}
                 {data && data.length === 0 && <NoDataFoundRow />}
               </Table.Body>
+              <tfoot className={styles.footer}>
+                <Table.Row>
+                  <Table.Cell colSpan={1000}>
+                    <div className={styles.footerLeft}>{action}</div>
+                    <div className={styles.footerRight}>
+                      {isFirstPageLoaded &&
+                        totalLength &&
+                        `Showing 1 - ${formatNumber(size)} of ${formatNumber(
```

Comment:
I used [thin space](http://jkorpela.fi/chars/spaces.html) there but I am not sure in this.
```suggestion
                        // thin spaces between numbers
                        `Showing 1\u2009-\u2009${formatNumber(size)} of ${formatNumber(
```